README.md

```diff
@@ -283,10 +283,36 @@ And finally, you are ready to go! Or at least you should be...
 
 ### Junior Contributors
 
+**Gemma Sagastegui** | [<gemma.sagastegui@alum.esci.upf.edu>][sagggastegui] | CoffeeL. 
 
+[sagggastegui] : <https://github.com/deliaBlue>
```

Comment:
This should be a link to your github account. please change it and put it in the corresponding line with the other links (remember to do it alphabetically)